### DISCUSSION TITLE
🏈 Remastered

### DISCUSSION THREAD

**Discussion by swiftyfinch** [swiftyfinch/Rugby#230] [Releases]
<p align="center">
  <img src="https://user-images.githubusercontent.com/64660122/230763146-d467dafb-252c-46ce-93d6-f2309e5aabc8.gif" width=600>
</p>

# 🏈 Rugby: Remastered has been finally released 🥳 

I've published [a new post in my blog about this](https://swiftyfinch.github.io/en/2023-04-22-rugby-remastered/).
Thanks a lot to everyone who used the beta versions! There were 14 125 usage count 🚀 

Use this command to update Rugby from `2.0.0b2+`:
```sh
> rugby update
```
Or read the guide 🦮 [how to install](https://github.com/swiftyfinch/Rugby/blob/2.0.0/docs/how-to-install.md) it.
Check out the 📖 [readme](https://github.com/swiftyfinch/Rugby/blob/2.0.0). You can find there all information.

## Changelog

- 🐛 Fix the bug with the 12-hours format;
- 🏛️ Return the `arch: auto` option and made it as default one.\
         Now, the `auto` means getting an architecture of the mac CPU instead of the current process;
- 👨🏻‍🔧 Added some changes in getting env. Now, you can use `rugby env` to find out Rugby environment:

```sh
> rugby env
Rugby version: 2.0.0
Swift: 5.7.1
CLT: Xcode 14.1 (Build version 14B47b)
CPU: Apple M1 Max (arm64)
Project: Unknown
Git branch: main
```

<br>

<hr /><em>This discussion was created from the release <a href='https://github.com/swiftyfinch/Rugby/releases/tag/2.0.0'>🏈 Remastered</a>.</em>

**Comment by Lutzifer** [comment]
Thanks for all the work on V2.0

Sadly, the way updates work now (rolling your own update mechanism) prohibits using rugby in CI Environments (at least for me).

**Reply by swiftyfinch** [reply]
@Lutzifer Thank you for you feedback!

I think that it's better to use just `curl` and `unzip` commands without any package manager.
It should be suited for any CI Environments. Please, check [the first install section](https://github.com/swiftyfinch/Rugby/blob/2.0.0/docs/how-to-install.md#first-install-zsh).

And I will add an example how to install 🏈 Rugby in CI and will use it in GitHub Actions.
https://github.com/swiftyfinch/Rugby/blob/main/.github/workflows/regress.yml
Thanks again! I think it will be useful for whole community.

**Reply by swiftyfinch** [reply]
@Lutzifer I've just added an example:
https://github.com/swiftyfinch/Rugby/blob/main/docs/how-to-install.md#install-in-ci-environment
https://github.com/swiftyfinch/Rugby/actions/runs/4790694440
